operations/mimir-mixin/dashboards/writes.libsonnet

```diff
@@ -370,56 +370,58 @@ local filename = 'mimir-writes.json';
         $.panelDescription(
-          'Replicas (leader zone)',
+          'Replicas (HPA + ReplicaTemplate)',
           |||
-            The minimum, maximum, and current number of replicas for the leader zone of ingesters.
-            Other zones scale to follow this zone (with delay for downscale).
+            The minimum, maximum, and current number of replicas reported by the HPA for the ReplicaTemplate object.
+            If available, also the spec and status replicas fields for the ReplicaTemplate object itself.
+            Rollout-operator will keep ingester replicas updated based on the ReplicaTemplate spec field, and then update the template's status field once the ingester count changes.
           |||
         )
       )
       .addPanel(
-        $.timeseriesPanel('Replicas') +
-        $.panelDescription('Replicas', 'Number of ingester replicas per zone.') +
-        $.queryPanel(
-          [
-            'sum by (%s) (up{%s})' % [$._config.per_job_label, $.jobMatcher($._config.job_names.ingester)],
-          ],
-          [
-            '{{ %(per_job_label)s }}' % $._config.per_job_label,
-          ],
-        ),
-      )
-      .addPanel(
-        $.autoScalingDesiredReplicasByValueScalingMetricPanel('ingester', '', '') + { title: 'Desired replicas (leader zone)' }
-      )
-      .addPanel(
-        $.autoScalingFailuresPanel('ingester') + { title: 'Autoscaler failures rate' }
-      ),
-    )
-    .addRowIf(
-      $._config.show_ingest_storage_panels && $._config.autoscaling.ingester.enabled,
-      $.row('Ingester – autoscaling (ingest storage)')
-      .addPanel(
-        $.autoScalingActualReplicas('ingester') + { title: 'Replicas (ReplicaTemplate)' } +
+        $.timeseriesPanel('Replicas (Ingesters)') +
         $.panelDescription(
-          'Replicas (ReplicaTemplate)',
+          'Replicas (Ingesters)',
           |||
-            The minimum, maximum, and current number of replicas for the ReplicaTemplate object.
-            Rollout-operator will keep ingester replicas updated based on this object.
+            Number of up ingester replicas per zone.
+            Also show the number of read-only replicas per zone, or number of Inactive partitions for ingest storage.
           |||
-        )
-      )
-      .addPanel(
-        $.timeseriesPanel('Replicas') +
-        $.panelDescription('Replicas', 'Number of ingester replicas.') +
-        $.queryPanel(
+        ) + $.queryPanel(
           [
             'sum by (%s) (up{%s})' % [$._config.per_job_label, $.jobMatcher($._config.job_names.ingester)],
+            'sum by (%s) (cortex_lifecycler_read_only{%s}) unless on (%s) (cortex_partition_ring_partitions{name="ingester-partitions"})' % [$._config.per_job_label, $.jobMatcher($._config.job_names.ingester), $._config.per_job_label],  // TODO: need to not show this if using ingest storage
```

Comment:
What's about this TODO? Isn't addressed by the `unless`?

Reply:
🤦 i just forgot to clean this up. thank you.